README.md

```diff
@@ -68,30 +81,13 @@ When installing using the Helm chart, you can choose to use a specific image tag
 
 Get started quickly with the Kubernetes Operator for Apache Spark using the [Quick Start Guide](docs/quick-start-guide.md).
 
-If you are running the Kubernetes Operator for Apache Spark on Google Kubernetes Engine and want to use Google Cloud Storage (GCS) and/or BigQuery for reading/writing data, also refer to the [GCP guide](docs/gcp.md).
-
```

Comment:
Isn't this part still relevant?

Reply:
I noticed that the GCP documentation is still in the docs folder, so I will undo the changes that removed it.